google-cloud-spanner/lib/google/cloud/spanner/admin/instance/v1/instance_admin_client.rb

```diff
@@ -204,6 +204,7 @@ def initialize \
                   timeout: timeout,
```

Comment:
Shouldn't service_address and service_port be passed in here as well?

Reply:
@dazuma? Too late for this PR, but feel free to make the changes to generate another.